docs/reference/upload-rights.rst

```diff
@@ -0,0 +1,120 @@
+Upload Rights
+#############
+
+Those who are allowed to upload the kernel to the Ubuntu archive take their
+responsibility very seriously.
+
+To obtain per package upload rights for the Ubuntu kernel, you can apply to
+become a member of the ubuntu-kernel-uploaders_ team in Launchpad.
+
+People can join this team only after going through a thorough application and
+review process. The sections below describe the general member profile of the
+ubuntu-kernel-uploaders_ team, as well as the application process.
```

Comment:
```suggestion
To obtain per package upload rights for the Ubuntu kernel, you can apply to
become a member of the ubuntu-kernel-uploaders_ team in Launchpad.
People can join this team only after going through a thorough application and
review process.

The sections below describe the general member profile of the
ubuntu-kernel-uploaders_ team, as well as the application process.
```

I suggest changing the structure slightly. The 1st paragraph is an intro to the topic. The 2nd paragraph provides a bit more context about the topic. The 3rd paragraph lets the user know the scope of what is covered about the topic in the current reference doc.